yb-voyager/cmd/config_test.go

```diff
@@ -64,6 +67,7 @@ func resetFlags(cmd *cobra.Command) {
 		f.Value.Set(f.DefValue)
 		f.Changed = false
 	})
+	utils.ErrExit = originalErrExit
```

Comment:
Are we sure `resetCmdAndEnvVars` and `resetFlags` are used together can we merge these functions, to also ensure that we change and restore the errExit in a single place?

Reply:
Yes, they are. one is called at the beginning of the test, and the other at the end of the test.
However, that's a good point. I'm thinking of putting it in every test case as opposed to in helper; makes things explicit, and easier to catch wrong usage.

Reply:
done ^